sdk/scheduler/src/main/java/com/mesosphere/sdk/api/PlansResource.java

```diff
@@ -22,6 +24,7 @@
     static final Response PLAN_ELEMENT_NOT_FOUND_RESPONSE = Response.status(Response.Status.NOT_FOUND)
             .entity("Element not found")
             .build();
+    private static final StringMatcher ENVVAR_MATCHER = RegexMatcher.create("[A-Za-z_][A-Za-z0-9_]*");
```

Comment:
I know we already have this somewhere, but a cursory search didn't reveal it, so...#lazyweb